### DISCUSSION TITLE
Profile Achievements

### DISCUSSION THREAD

**Discussion by eronaralves** [community/community#18938] [Other Feature Feedback, Questions, & Ideas] [answered]
Is there any documentation explaining how to unlock achievements?

**Comment by JonasJs** [comment, marked as answer]
There is this repository: https://github.com/Schweinepriester/github-profile-achievements/

**Reply by alenworld** [reply]
Thank you!

**Comment by neriidesu** [comment]
There is this repository: https://github.com/Schweinepriester/github-profile-achievements/

**Comment by talregev** [comment]
I wish you show the full list of the repository for Arctic Code Vault Contributor.
Also I really like you doing the profile achievement. Very engagement.

**Comment by janapc** [comment]
Thank you for your help

**Reply by YouthguyZ** [reply]
> Thank you for your help
hey follow me it's so funny🤠

**Comment by 4Evens** [comment]
There is this repository: https://github.com/Schweinepriester/github-profile-achievements/

**Comment by gyt95** [comment]
Exactly what I need, thanks.

repository: https://github.com/Schweinepriester/github-profile-achievements/

**Comment by AlbinBee** [comment]
Galaxy Brain <==> 2 accepted answers
Pull Shark <==> 2 pull requests merged
YOLO <==> Merged pull request without code review
